src/main/java/seedu/dietmanager/storage/Storage.java

```diff
@@ -117,6 +127,121 @@ public void loadDataDirectory() {
         }
     }
 
+    /**
+     * Searches for the data file in the directory, if absent, creates a new data file. <br>
+     * If data file is present, loads the existing data from the file such that it is accessible by the user.
+     */
+
+    public void loadRecipeFile() {
+        try {
+            File recipeData = new File(RECIPE_FILE_PATH);
+            if (recipeData.createNewFile()) {
+                logsCentre.writeInfoLog("No existing Recipe file found, new file created: "
+                        + recipeData.getName());
+            } else {
+                logsCentre.writeInfoLog("Existing Recipe file found: "
+                        + recipeData.getName());
+                this.readRecipeFile();
+            }
+        } catch (IOException e) {
+            logsCentre.writeSevereLog("Error in Recipe data file");
+            ui.displayFileErrorMessage();
+        }
+    }
+
```

Comment:
Writing of recipe looks good. Storage function seems to work but require more testing